xcat3/common/rest.py

```diff
@@ -0,0 +1,45 @@
+from oslo_log import log as logging
+from xcat3.common import exception
+
+import requests
+import json
+
+LOG = logging.getLogger(__name__)
+
+headers = {'Content-Type': 'application/json'}
+
+def login (url, in_data) :
```

Comment:
I'm not sure if you should add login method here as login is not a common operation for rest